### PR TITLE
Add test file

### PR DESCRIPTION
**Pull request by vankatalp360**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,356 @@
+
```
Comment (MDaskalovAtBE) [inline .gitignore]: has that file been intended for adding

**Review by MDaskalovAtBE — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (vankatalp360) [reply .gitignore]: yes

**Comment by vankatalp360** [comment]
Yes

File: .gitignore (hunk 1 above)
Comment (MDaskalovAtBE) [reply .gitignore]: just for ur learning skills could u please delete it and commit the change as at the end of the day I will decline that PR anyway...

**Pull request closed without merging**